### PR TITLE
Feature/metadata

### PR DESCRIPTION
**Pull request by robin-smileid**
Story: https://app.shortcut.com/smileid/story/15105/add-fraud-signals-metadata-to-the-sdk-android

## Summary

A few sentences/bullet points about the changes

## Known Issues

Any shortcomings in your work. This may include corner cases not correctly handled or issues related
to but not within the scope of your PR. Design compromises should be discussed here if they were not
already discussed above.

## Test Instructions

Concise test instructions on how to verify that your feature works as intended. This should include
changes to the development environment (if applicable) and all commands needed to run your work.

## Screenshot

If applicable (e.g. UI changes), add screenshots to help explain your work.

### CODE REVIEW

File: lib/src/main/java/com/smileidentity/compose/metadata/updaters/CameraInfoProvider.kt [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+ * A manager that updates metadata with camera information.
+ */
+internal class CameraInfoProvider(
+    context: Context,
+    private val metadata: SnapshotStateList<Metadatum>,
+) : MetadataInterface {
+
+    private val cameraManager = context.getSystemService(Context.CAMERA_SERVICE) as CameraManager?
+
+    private fun getNumberOfCameras(): Any {
```
Comment (jumaallan) [inline lib/src/main/java/com/smileidentity/compose/metadata/updaters/CameraInfoProvider.kt:19]: the size returns a count which is an an int but you fallback to a string since you use any

might be a problem typecasting the types back

**Comment by robin-smileid** [comment]
Closing in favour of https://github.com/smileidentity/android/pull/605
